cookbook/security/remember_me.rst

```diff
@@ -8,10 +8,7 @@ Once a user is authenticated, their credentials are typically stored in the
 session. This means that when the session ends they will be logged out and
 have to provide their login details again next time they wish to access the
 application. You can allow users to choose to stay logged in for longer than
-the session lasts using a cookie with the ``remember_me`` firewall option.
-The firewall needs to have a secret key configured, which is used to encrypt
```

Comment:
Thanks for reporting this error. I totally forgot to add the `key` option to the list. I've added it in 7e45958